src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/IsNullOrEmptyConverter.shared.cs

```diff
@@ -18,10 +24,25 @@ public class IsNullOrEmptyConverter : ValueConverterExtension, IValueConverter
 		/// <param name="parameter">Additional parameter for the converter to handle. This is not implemented.</param>
 		/// <param name="culture">The culture to use in the converter. This is not implemented.</param>
 		/// <returns>A <see cref="bool"/> indicating if the incoming value is null or empty.</returns>
-		public object Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture) => ConvertInternal(value);
+		public object Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture)
+			=> InvertCheck != ConvertInternal(value);
```

Comment:
It's not obvious (at least to me) that `InvertCheck !=` inverts the result of `ConvertInternal`. I think inversion should be written more clearly for better readability.

```suggestion
		{
			var result = ConvertInternal(value);
			if (InvertCheck)
				result = !result;
			
			return result;
		}
```

Reply:
Done!